pythonforandroid/bootstraps/sdl2/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -54,7 +54,7 @@
     <application android:label="@string/app_name"
                  android:icon="@drawable/icon"
                  android:allowBackup="{{ args.allow_backup }}"
-                 android:theme="@android:style/Theme.NoTitleBar{% if not args.window %}.Fullscreen{% endif %}"
+                 android:theme="{{args.android_apptheme}}{% if not args.window %}.Fullscreen{% endif %}"
```

Comment:
Mildly awkward here that only the sdl2 AndroidManifest is changed - I guess this should be implemented in every bootstrap?

Reply:
i did that now also for the other bootstraps